pkgs/applications/editors/gnome-latex/default.nix

```diff
@@ -0,0 +1,47 @@
+{ stdenv, fetchurl, wrapGAppsHook
+, tepl, amtk, gnome3, glib, pkgconfig, intltool, itstool, libxml2 }:
+let
+  version = "3.28.1";
+  pname = "gnome-latex";
+in stdenv.mkDerivation {
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    url = "mirror://gnome/sources/${pname}/${gnome3.versionBranch version}/${pname}-${version}.tar.xz";
+    sha256 = "1z481izrx057wraphnr82kxnpmmi8nvl7jswyylzm22kfs0mw402";
+  };
+
+  NIX_CFLAGS_COMPILE = "-I${glib.dev}/include/gio-unix-2.0";
+
+  nativeBuildInputs = [
+    pkgconfig
+    wrapGAppsHook
+    itstool
+    intltool
+  ];
+
+  buildInputs = with gnome3; [
+    amtk
+    dconf.dev
```

Comment:
Why are you specifically requesting `dconf.dev`? Actually, we should probably build with `--disable-dconf-migration` as we did not have the package before so there are no old GSettings paths to migrate.